src/UI/Seller/src/app/products/components/product-edit/product-edit.component.ts

```diff
@@ -205,6 +208,8 @@ export class ProductEditComponent implements OnInit, OnDestroy {
       this.addresses = await this.ocSupplierAddressService
         .List(context.Me.Supplier.ID)
         .toPromise()
+    } else if (context.Me.Seller) {
+      this.addresses = await this.ocAdminAddressService.List().toPromise()
```

Comment:
Can you use the javascript sdk equivalent? To access you can do:

```typescript
import { AdminService } from 'ordercloud-javascript-sdk';
```

Goal is to eventually remove our dependence on the angular sdk so that we only interact with one SDK, the javascript SDK. If you feel up to it, would be great to refactor the other services to use the javascript one. If not, would you mind just adding a TODO?

Reply:
Updated